devops/db/liquibase/changelogs/payment-transaction-gateway/scripts/table-creation/V2__PM_739_PP_PGS_REQUEST_INFO_DEFINITION.sql

```diff
@@ -13,6 +13,7 @@ CREATE TABLE PP_PGS_REQUEST_INFO (
   IS_PROCESSED      NUMBER,
   MDC_INFO          VARCHAR2(500 CHAR),
   ERROR_CODE        VARCHAR2(50 CHAR),
+  STATUS            VARCHAR2(50 CHAR),
```

Comment:
@andrearovere you should use liquibase changelogs logic to make DB changes effective in SIT on cloud.
Please, refactor this as an "add column" statement in a new separate sql file and use changelog files